tasties_app/tests/conftest.py

```diff
@@ -183,3 +184,12 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def salad_recipes_with_rating():
+    recipes_list = Recipe.objects.filter(title__icontains='salad')
+    recipes_with_ratings = {}
+    for recipe in recipes_list:
+        recipes_with_ratings[recipe] = Rating.objects.filter(recipe_id=recipe).aggregate(Avg('rating'))['rating__avg']
```

Comment:
Why do you need a dictionary with the rating values here?
Just send the recipes list filtered by 'salad'.